crates/matrix-sdk-ui/src/timeline/controller/metadata.rs

```diff
@@ -0,0 +1,361 @@
+// Copyright 2023 The Matrix.org Foundation C.I.C.
```

Comment:
!!! Did you manage to craft a Delorean??

(nit: 2025)

Reply:
Fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/4595, thanks @bnjbvr!